provider/azure/azure_private_dns.go

```diff
@@ -58,16 +58,18 @@ type AzurePrivateDNSProvider struct {
 	zonesClient                  PrivateZonesClient
 	zonesCache                   *zonesCache[privatedns.PrivateZone]
 	recordSetsClient             PrivateRecordSetsClient
+	MaxRetriesCount              int
```

Comment:
```suggestion
	maxRetriesCount              int
```
Would you please follow golang standard on variable naming ?

Reply:
Sure @mloiseleur, changed the variable according to golang standard